audio/VGGish_Audioset_&_Audio_embedding_Tutorial.ipynb

```diff
@@ -0,0 +1,2710 @@
+{
```

Comment:
Pushing Python Notebook with all outputs makes future diff and code review quite hard.
Minimal requirement would be to manually always clear all outputs before pushing to repo.
This link talks about how to automate this: http://timstaley.co.uk/posts/making-git-and-jupyter-notebooks-play-nice/
Additional useful option would be to have "save hook" that automatically saves  python-only script ver of the notebook (for future easier code reviews) as well as html ver (so it is possible to see the outputs), see ideas here https://www.svds.com/jupyter-notebook-best-practices-for-data-science/

Reply:
Great learning, thanks for the best practices!